niworkflows/utils/spaces.py

```diff
@@ -360,10 +377,15 @@ class SpatialReferences:
       ...
     ValueError: space ...
 
+    >>> sp.cache_spaces()
+    Traceback (most recent call last):
+     ...
+    AttributeError: Spaces have already ...
+
     """
 
-    __slots__ = ('_spaces',)
-    standard_spaces = tuple(_tfapi.templates())
+    __slots__ = ('_spaces', '_cached')
+    _standard_spaces = tuple(_tfapi.templates())
```

Comment:
I'd rather keep this public.
```suggestion
    standard_spaces = tuple(_tfapi.templates())
```